src/main/java/pages/createAccountPage/CreateAccountPage.java

```diff
@@ -0,0 +1,101 @@
+package pages.createAccountPage;
+
+import lombok.Getter;
+import lombok.Setter;
+import models.AccountModel;
+import models.CustomerModel;
+import org.openqa.selenium.By;
+import pages.BasePage;
+
+@Getter
+@Setter
+public class CreateAccountPage {
+    protected BasePage basePage;
+    public AccountModel accountModel;
+    public CustomerModel customerModel;
+
+    public String alertMessageXpath = "//div[contains(@class, 'alert-error')]";
+    public By alertMessageLocator = By.xpath(alertMessageXpath);
+    public By alertMessageCloseButtonLocator = By.xpath(alertMessageXpath + "//button[@class='close']");
+    public By continueButtonLocator = By.xpath("//button[@title='Continue']");
+
+    public CreateAccountPage() {
+        this.basePage = new BasePage();
+        this.accountModel = new AccountModel();
+        this.customerModel = accountModel.getCustomerModel();
+    }
+
+    public CreateAccountPage(CustomerModel customerModel, Boolean isSubscribedToNewsletter) {
+        this.basePage = new BasePage();
+        this.accountModel = new AccountModel(customerModel, isSubscribedToNewsletter);
+        this.customerModel = customerModel;
+    }
+
+    public By inputFormLocator(CreateAccountInput input) {
```

Comment:
If you want to do it like that please take those method to separate class:)

Reply:
Done. I created new class ```AccountFormUtils``` to manage the form associated with creating account:
- Moved all methods (like ```getHelpTextForInput```, ```selectStateOrCountry```, ```acceptPrivacyPolicy```, ```fillInput```) to ```AccountFormUtils```
- Moved ```fillCreateAccountForm(AccountModel accountModel)``` to ```CreateAccountPageSteps```
- ```CreateAccountPage``` class has methods related only to elements on the page